test/verify/check-apps

```diff
@@ -82,6 +86,12 @@ class TestApps(PackageCase):
         self.createAppStreamPackage("app-1", "1.0", "1")
         self.enableAppStreamRepo()
 
+        b.wait_present(".app-list tr:contains('An application for testing')")
+        b.click(".app-list tr:contains('An application for testing')")
+        b.wait_present(".app-links div:contains('<a href=\"https://app-1.com\" target=\"_blank\" rel=\"noopener\" data-linkedhost=\"https://app-1.com\">Homepage</a>')")
```

Comment:
I would use a jquery style selector here. Not the full html text.

So something like ```b.wait_present('.app-links a[href="https://app-1.com"]')```